ibm_mq/datadog_checks/ibm_mq/ibm_mq.py

```diff
@@ -129,3 +144,15 @@ def send_metrics_from_properties(self, properties, metrics_map, prefix, tags):
                     )
                     return
                 self.send_metric(metric_type, metric_full_name, metric_value, new_tags)
+
+    def check_process_indicator(self):
```

Comment:
I don't like this name much, what about `check_queue_manager_process`

Reply:
The new condition limiting in the base class uses a more efficient way where when the limit is reached no conditions are even evaluated (i.e. no psutil) until an earlier slot is explicitly released. I could add caching to the brief stage before a process is found but I think it's not worth it